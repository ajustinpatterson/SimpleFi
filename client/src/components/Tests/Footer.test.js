import React from "react";
import renderer from "react-test-renderer"
import Footer from "../Footer/Footer";
import {
  act,
  render,
  wait,
  fireEvent,
  screen
} from "@testing-library/react";
import { BrowserRouter } from 'react-router-dom';


it ("renders correctly", () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();

});