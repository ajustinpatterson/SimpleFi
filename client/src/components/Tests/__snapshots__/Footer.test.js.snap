// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer>
  <span>
    © 2020 SimpleFi
  </span>
  <span>
    ·
  </span>
  <span>
    Privacy Policy
  </span>
  <span>
    ·
  </span>
  <span>
    Twitter
  </span>
  <span>
    ·
  </span>
  <span>
    Telegram
  </span>
</footer>
`;
